python/pdstools/adm/ADMDatamart.py

```diff
@@ -24,6 +22,9 @@
 from .ADMTrees import ADMTrees
 from .Tables import Tables
 
+logger = logging.getLogger("PDSTools.ADMDatamart")
+logging.basicConfig(level=logging.INFO)
```

Comment:
I don't think you need this

Reply:
I am logging the quarto outputs at INFO level
```{python}
process = subprocess.Popen(
            command,
            stdout=subprocess.PIPE,
            stderr=subprocess.PIPE,
            cwd=temp_dir,
            text=True,
        )

while True:
    output = process.stdout.readline()
    error = process.stderr.readline()
    if output:
        logger.info(output.strip())
    if error:
        logger.info(error.strip())
    if process.poll() is not None and not output and not error:
        break
```

Reply:
Sure - but you shouldn't need to use basicConfig configuration in the logging _writing_ scripts, only the consuming? To write as info logging, simply do `logger.info()`. Unless I'm missing something?